thoth/storages/graph/postgres.py

```diff
@@ -2889,6 +2889,10 @@ def get_software_environments_all(
         python_version: Optional[str] = None,
         cuda_version: Optional[str] = None,
         image_name: Optional[str] = None,
+        library_name: Optional[str] = None,
+        symbol: Optional[str] = None,
+        package_name: Optional[str] = None,
+        rpm_package: Optional[str] = None,
```

Comment:
what about `rpm_package_name` to be more clear about the input?

Reply:
Done (here and in https://github.com/thoth-station/user-api/pull/1628)